docs/source/integrations/mlflow.rst

```diff
@@ -1,3 +1,148 @@
 ======
-MLFlow
-======
\ No newline at end of file
+MLflow
+======
+
+MLflow for training and experimentation,
+BentoML for production serving and deployment,
+They work together nicely
+
+Save model natively with BentoML
+--------------------------------
+
+    1. save with bentoml next to log_model
+
+        mlflow.sklearn.log_model(..)
+        bentoml.sklearn.save_model(..)
+
+    2. load_model(model_uri) and then save with BentoML
+
+        model = mlflow.sklearn.load_model(model_uri)
+        bentoml.sklearn.save_model("..", model)
+
+    Note: Recommended using this approach if need GPU inference
+
+Import MLflow model to BentoML
+------------------------------
+
+    bentoml.mlflow.import_model("model_name", model_uri=...)
+
+About model_uri:
+
+    URI to the model. A local path, a 'runs:/' URI, or
+    a remote storage URI (e.g., an 's3://' URI). For
+    more information about supported remote URIs for
+    model artifacts, see https://mlflow.org/docs/latest
+    /tracking.html#artifact-stores  [required]
+
+    The location, in URI format, of the MLflow model, for example:
+
+    /Users/me/path/to/local/model
+    relative/path/to/local/model
+    s3://my_bucket/path/to/model
+    runs:/<mlflow_run_id>/run-relative/path/to/model
+    models:/<model_name>/<model_version>
+    models:/<model_name>/<stage>
+
+
+Load MLflow model saved with BentoML
+------------------------------------
+
+* Load with original flavor
+    > bento_model = bentoml.mlflow.get("..")
+    > pytorch_model = mlflow.pytorch.load_model(bento_model.path_of("mlflow_model"))
+
+* Load pyfunc model
+    > bento_model: PyfuncModel = bentoml.mlflow.load_model("..")
+
+* Using MLFlow Runner
+    > runner = bentoml.mlflow.get('my_model:latest').to_runner()
```

Comment:
it would be nice if underlying runner mechanism we return the correct framework runner. Maybe we can add a special `_use_framework_runner=True` to `get`. The default behaviour would be `False` and use `mlflow.pyfunc.PyFunc` runner.

Reply:
Users can already do that by loading the model and save it with bentoml again, as documented in the first section in the docs. I don't think we should do this automatically out of the box tho, there's no reliable way to make it work consistently